packages/dom/src/dom/is-html-input-element.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable jsdoc/valid-types */
```

Comment:
Why is it disabled here and enabled below?

Reply:
Because the `element is HTMLInputElement` syntax isn't considered a "valid type" by the JSDoc parser (doctrine) that the eslint JSDoc rules use. If you search the codebase you'll see that we've unfortunately had to disable these all over the place.